sessionctx/variable/varsutil.go

```diff
@@ -488,9 +488,15 @@ func parseTimeZone(s string) (*time.Location, error) {
 	}
 
 	// The value can be given as a string indicating an offset from UTC, such as '+10:00' or '-6:00'.
+	// Hour should not greater than 12, like '+13:00', or '-13:00' is invalid.
 	if strings.HasPrefix(s, "+") || strings.HasPrefix(s, "-") {
 		d, err := types.ParseDuration(nil, s[1:], 0)
 		if err == nil {
+			hofst := int(d.Duration / time.Hour)
+			if hofst > 12 {
```

Comment:
In fact, Time zone [TKT](https://en.wikipedia.org/wiki/Time_zone), is UTC+13, MySQL's limit is [-12:59,+13:00]

Reply:
you are right, PTAL again